prototype/README

```diff
@@ -0,0 +1,7 @@
+Compiling and running the prototype (until we have a proper setup)
+
+1. Compile Dawn
+2. cd build && ./bin/unittest/DawnCUnittest --gtest_also_run_disabled_tests 2>../prototype/generated.hpp && clang-format -i ../prototype/generated.hpp
```

Comment:
```suggestion
2. cd <path/to/dawn> && ./bundle/build/dawn-prefix/src/dawn-build/bin/unittest/DawnCUnittest --gtest_also_run_disabled_tests 2>./prototype/generated.hpp && clang-format -i ./prototype/generated.hpp
```

Reply:
hmm didn't thought about people using the bundle... but I'm fine with this path.